js/popover.js

```diff
@@ -65,10 +65,16 @@
     var $e = this.$element
     var o  = this.options
 
-    return $e.attr('data-content')
+    var content = $e.attr('data-content')
       || (typeof o.content == 'function' ?
             o.content.call($e[0]) :
             o.content)
+
+    if (['[object Function]', '[object Object]', '[object String]'].indexOf(Object.prototype.toString.call(content)) === -1) {
```

Comment:
@hnrch02 when setting the variable content in this, I get the value of the function as `content` not the function itself. Thus, in the tests I end up with a jQuery object from items like:

```js
content: function () {
          return $div
        }
```